### PR TITLE
Update README.md: HTerrain supports Godot 4.1

### PR DESCRIPTION
**Pull request by MJacred**

### CODE REVIEW

**Review by skyace65 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 - [GodotTIE](https://github.com/henriquelalves/GodotTIE) - Text Interface Engine to control text output (like in a RPG dialogue) (Godot 2.x, 3.x).
 - [gterm](https://github.com/TeddyDD/gterm) - GUI control that draws something similar to \*nix terminal emulators (Godot >= 2.1).
 - [GUT](https://github.com/bitwes/Gut) - Utility for writing unit tests in GDScript (Godot 3.x).
-- [HTerrain](https://github.com/Zylann/godot_heightmap_plugin) - Heightmap-based terrain. Supports texture painting, colouring, holes, level of detail and grass (Godot 3.x).
+- [HTerrain](https://github.com/Zylann/godot_heightmap_plugin) - Heightmap-based terrain. Supports texture painting, colouring, holes, level of detail and grass (Godot 3.x, 4.1).
```
Comment (Calinou) [inline README.md]:
```suggestion
- [HTerrain](https://github.com/Zylann/godot_heightmap_plugin) - Heightmap-based terrain. Supports texture painting, colouring, holes, level of detail and grass (Godot 3.x, 4.x).
```

**Comment by Calinou** [comment]
Since the add-on should work well with 4.2 (or will start doing so soon), I'll mark it as `4.x`.

**Comment by MJacred** [comment]
@Calinou: afaik it does not work with 4.0, only starting 4.1

Official docs say `You will need to use Godot 4.1 or later.`: https://github.com/Zylann/godot_heightmap_plugin/blob/master/addons/zylann.hterrain/doc/docs/index.md#how-to-install

**Comment by Calinou** [comment]
> @Calinou: afaik it does not work with 4.0, only starting 4.1
> 
> Official docs say `You will need to use Godot 4.1 or later.`: [Zylann/godot_heightmap_plugin@`master`/addons/zylann.hterrain/doc/docs/index.md#how-to-install](https://github.com/Zylann/godot_heightmap_plugin/blob/master/addons/zylann.hterrain/doc/docs/index.md?rgh-link-date=2023-12-11T11%3A24%3A50Z#how-to-install)

Yes, but 4.0 is pretty much EOL at this point. Most users have moved on to 4.1 at least by now :slightly_smiling_face: 

It's like "Godot 3.x": most add-ons on that version don't support versions below 3.5 or 3.4.

**Comment by MJacred** [comment]
Alrighty :)

**Pull request merged**